src/spatial_ref/srs.rs

```diff
@@ -284,34 +284,25 @@ impl SpatialRef {
     }
 
     #[cfg(major_ge_3)]
-    pub fn name(&self) -> Result<String> {
+    pub fn name(&self) -> Option<String> {
         let c_ptr = unsafe { gdal_sys::OSRGetName(self.0) };
```

Comment:
`c_ptr` is onwed by self, so we don't need to free it?

Reply:
Owned by PROJ via `OGRSpatialReference`.